plone/protect/views.py

```diff
@@ -4,6 +4,17 @@
 from zExceptions import Forbidden
 from zope.interface import implementer
 
+import warnings
+
+
+warnings.warn(
+    "You are using the base ConfirmView view in plone.protect,"
+    " for classic UI, override the ConfirmView from plone.app.layout"
+    " by subclassing your BrowserLayer of"
+    " plone.app.layout.interfaces.IPloneAppLayoutLayer.",
+    DeprecationWarning,
+)
+
```

Comment:
I would stick using this class, see also:

- https://github.com/plone/plone.app.layout/pull/386#discussion_r1989019037
- https://github.com/plone/plone.app.layout/pull/386#discussion_r1989021246

We can always move it later.